save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/WelcomeView.kt

```diff
@@ -266,14 +264,13 @@ class WelcomeView : AbstractView<WelcomeProps, IndexViewState>(true) {
             className = ClassName("card-body")
             div {
                 className = ClassName("text-sm")
-
-                menuTextAndLink("Contests", "/#/$CONTESTS", faBell)
+                menuTextAndLink("Contests", "/#/${FrontendRoutes.CONTESTS.path}", faBell)
                 hrNoMargin()
-                menuTextAndLink("List of Projects", "#/$PROJECTS", faExternalLinkAlt)
+                menuTextAndLink("List of Projects", "#/${FrontendRoutes.PROJECTS.path}", faExternalLinkAlt)
```

Comment:
to tell you the truth - I would have made it shorter with importing FrontendRoutes, but that is fine by for now...